src/share.rs

```diff
@@ -448,6 +448,17 @@ impl fmt::Debug for RecvStream {
     }
 }
 
+impl Drop for RecvStream {
+    fn drop(&mut self) {
+        // Eagerly clear any received DATA frames now, since its no longer
+        // possible to retrieve them. However, this will be called
```

Comment:
is _this_ `drop` or `clear_recv_buffer`? I assume the latter, but it's not clear

Reply:
The call to `clear_recv_buffer`.